converter/converter.go

```diff
@@ -71,12 +78,25 @@ func toYAML(jsonTemplate []byte) ([]byte, error) {
 
 func toJSON(yamlTemplate []byte) ([]byte, error) {
```

Comment:
I believe that name of that function should reflect fact that we're converting from _YAML_.